.github/workflows/conventional-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: conventional-pr
```

Comment:
It's ok for now, but I'm a bit afraid about the complexity of maintaining a monorepo setup. For example, we enforce conventions only for app PRs, consider `[Release] ..` commits to refer to app releases but not CLI releases, and mix CLI and App releases.
I'd try to sketch what an iteration of our current architecture would look like to:
1. Support the development of some code in private
2. Support the development of the app in a different repository
3. Eliminate the assumption from the existing code that the consumer is a CLI.

Reply:
I'm leaning to a monorepo for the CLI and the app, at least for now, so we don't have to do too many cross-repo changes. If we were to move app into its own repo, we'd need to create an _extra_ repo for the common code across the CLI and the App. This is because App depends on `TuistSupport` and we don't want to get into a situation where we can't merge a PR in the `App` because we need to wait for a new release of `TuistSupport` that's in the CLI that has a weekly release cycle.

Reply:
We are aligned then. That comment was more toward the future to keep an eye on the complexity that can emerge from this setup.